jaxrs-api/src/main/java/se/fortnox/reactivewizard/jaxrs/FieldError.java

```diff
@@ -3,14 +3,18 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
+import java.io.Serializable;
 import java.util.Map;
 
-public class FieldError {
+public class FieldError implements Serializable {
```

Comment:
What benefit does this give?

Reply:
SonarCloud did not like that I added FieldError to a serializable class (DetailedError) without making it serializable or transient. Making it transient didn't feel right so I tried this instead but that just pushed the error into this class and the errorParams object instead :(

Reply:
SonarCloud also got mad that DetailedError is not immutable (it wants Throwables to be immutable) but it wasn't immutable before my changes either. What gives?